package.json

```diff
@@ -70,5 +61,16 @@
     "stylelint-order": "^0.8.1",
     "stylelint-scss": "^2.5.0",
     "yargs": "^11.1.0"
+  },
+  "dependencies": {
```

Comment:
these are still dev dependencies, not dependencies

Reply:
Nice catch, my bad, I seem to have installed without `--save-dev` flag.